style.css

```diff
@@ -144,3 +144,141 @@ figcaption {
 .author__name a:hover {
   color: rgb(108, 110, 38);
 }
+
+/* Smartphones (вертикальная и горизонтальная ориентация) ----------- */
+@media only screen and (max-width: 480px) {
+  .name__inf {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    background-color: aquamarine;
+  }
+  .name__inf h1 {
+    margin: 5px;
+    padding: 5px;
+    font-size: 30px;
+    margin-left: 60px;
+  }
+  .name__inf p {
+    margin: 30px;
+    padding: 5px;
+    font-size: 20px;
+  }
+
+  .basic__inf {
+    display: flex;
+    flex-direction: column;
+    background-color: rgb(250, 245, 245);
+  }
+  figure {
+    width: 100%;
+  }
+  figure img,
+  figcaption {
+    margin: 0 auto;
+  }
+
+  .contact {
+    width: 30%;
+    height: 100%;
+    display: flex;
+    letter-spacing: 2px;
+    flex-direction: column;
+    justify-content: space-evenly;
+  }
+  .contact__type {
+    font-size: 20px;
+    margin: 0 auto;
+  }
+
+  .main__inf {
+    flex-direction: column;
+  }
+  .main__inf h2 {
+    margin: 18px;
+    text-shadow: 2px 2px 3px gray;
+  }
+  .my_goal {
+    width: 50%;
+  }
+  .my_goal ol {
+    height: 60%;
+    width: 70%;
+    font-size: 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+  }
+  .my_skills {
+    width: 70%;
+    font-size: 20px;
+  }
+  .various__skills {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .experience {
+    height: 34%;
+    font-size: 20px;
+  }
+  .experience ul {
+    margin-left: 30px;
+  }
+  .experience ul li {
+    padding: 3px;
+  }
+  .experience ul li a {
+    text-decoration: none;
+  }
+  .experience ul li a:hover {
+    color: red;
+  }
+
+  .education {
+    height: 33%;
+    font-size: 20px;
+  }
+  .education p {
+    width: 70%;
+    padding: 20px;
+    line-height: 1.5;
+  }
+  .level_eng {
+    width: 70%;
+    font-size: 20px;
+    height: 33%;
+  }
+  .level_eng p {
+    padding: 20px;
+    line-height: 1.5;
+  }
+
+  .code__example {
+    width: 70%;
+  }
+  .code__example h2 {
+    width: 60%;
+  }
+  .code__example pre {
+    line-height: 1.5;
+    padding: 0px;
+  }
+  .author__name {
+    background-color: rgb(255, 235, 235);
+    display: flex;
+    flex-direction: column;
+  }
+  .author__name p {
+    margin: auto 80px;
+  }
+  .author__name a {
+    width: 20%;
+    text-decoration: none;
+    margin: auto 80px;
+  }
+  .author__name a:hover {
+    color: rgb(108, 110, 38);
+  }
```

Comment:
Inside media-queries should be only styles different from those styles, which are used by default.
Otherwise, you load a browser with huge work of parsing useless lines of styles, which is not good.